src/applications/vaos/utils/selectors.js

```diff
@@ -71,10 +70,14 @@ export const vaosExpressCareNew = state =>
 export const selectFeatureToggleLoading = state => toggleValues(state).loading;
 const vaosFlatFacilityPage = state =>
   toggleValues(state).vaOnlineSchedulingFlatFacilityPage;
+const vaosFlatFacilityPageSacramento = state =>
+  toggleValues(state).vaOnlineSchedulingFlatFacilityPageSacramento;
 export const selectUseFlatFacilityPage = state =>
   vaosFlatFacilityPage(state) &&
   !selectIsCernerPatient(state) &&
-  !selectIsRegisteredToSacramentoVA(state);
+  (!selectIsRegisteredToSacramentoVA(state) ||
```

Comment:
You can simplify this by doing `(!selectIsRegisteredToSacramentoVA(state) || vaosFlatFacilityPageSacramento(state))`. You don't need the extra `selectIsRegisteredToSacramentoVA` check, that has to be true to get to the logic after the `||`.